### PR TITLE
LDB_mix_data_cleanning

### PR DESCRIPTION
**Pull request by LDB-001**
My notebook with the cleaning script

### CODE REVIEW

File: ldb-eda-structure.py [hunk 1]
```diff
@@ -0,0 +1,505 @@
+#!/usr/bin/env python
```
Comment (yvesfracari) [inline ldb-eda-structure.py:1]: If this pull request it's about data cleanning this file shouldn't be here.

File: src/data/ldb-datacleaning.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (yvesfracari) [reply src/data/ldb-datacleaning.py:1]: I think you should change the name of the file to improve the documentation

File: notebooks/ldb-datacleaning.ipynb [hunk 3]
```diff
@@ -0,0 +1,1782 @@
+{
```
Comment (yvesfracari) [inline notebooks/ldb-datacleaning.ipynb:1]: <p>In the raw csv file, downloaded direct of the Kaggle, the sep is equal to ',' and decimal is equal to '.'.</p><p> </p><p>I think you change the csv file.</p><p> </p><p>I suggest to use:</p><p><code>train=pd.read_csv('../data/raw/train.csv')</code></p><p> </p><p>Also, you are importing the matplotlib package but you don't use it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (yvesfracari) [inline notebooks/ldb-datacleaning.ipynb:1]: <p>I think you could explain why this is in santd by and isnert a link of the issue!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: notebooks/ldb-datacleaning.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (LDB-001) [reply notebooks/ldb-datacleaning.py:1]: wich one? the .ipynb?

[the remark this answers is not in the document]
Comment (yvesfracari) [reply notebooks/ldb-datacleaning.py]: No, the .py automatically generated by the .ipynb

File: ldb-eda-structure.py (hunk 1 above)
Comment (LDB-001) [reply ldb-eda-structure.py]: Done!

File: notebooks/ldb-datacleaning.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (LDB-001) [reply notebooks/ldb-datacleaning.py]: Done

File: src/data/ldb-datacleaning.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (LDB-001) [reply src/data/ldb-datacleaning.py]: Done. I changed the comands related to the notebook and the analyses, also, I changed the name

File: notebooks/ldb-datacleaning.ipynb (hunk 3 above)
Comment (LDB-001) [reply notebooks/ldb-datacleaning.ipynb]: I removed the matplotlib plackage and I also changed the the script for dowloaded the csv file

**Pull request merged**